frontend/packages/knative-plugin/src/components/overview/__tests__/ConfigurationsOverviewListItem.spec.tsx

```diff
@@ -0,0 +1,41 @@
+import * as React from 'react';
+import { shallow } from 'enzyme';
+import { referenceForModel } from '@console/internal/module/k8s';
+import { ResourceLink } from '@console/internal/components/utils';
+import { ConfigurationModel } from '@console/knative-plugin';
+import { sampleKnativeConfigurations } from '@console/dev-console/src/components/topology/__tests__/topology-knative-test-data';
+import ConfigurationsOverviewListItem from '../ConfigurationsOverviewListItem';
+
+describe('ConfigurationsOverviewListItem', () => {
+  it('should list the Configuration', () => {
+    const wrapper = shallow(
+      <ConfigurationsOverviewListItem configuration={sampleKnativeConfigurations.data[0]} />,
+    );
+    expect(wrapper.type()).toBe('li');
+  });
+
+  it('should have ResourceLink with proper kind', () => {
+    const wrapper = shallow(
+      <ConfigurationsOverviewListItem configuration={sampleKnativeConfigurations.data[0]} />,
+    );
+    expect(wrapper.find(ResourceLink)).toHaveLength(1);
+    expect(
+      wrapper
+        .find(ResourceLink)
+        .at(0)
+        .props().kind,
+    ).toEqual(referenceForModel(ConfigurationModel));
```

Comment:
I've got mixed feelings about this test. It is properly testing the kind created is a proper link to ConfigurationModel but it's digging into the API of another component to do so.